### PR TITLE
Update dependabot to check rel/0.13

### PR DESCRIPTION
**Pull request by HTHou**
## Description

Update dependabot to check rel/0.13 branch.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -61,11 +61,11 @@ updates:
     directory: "/"
     schedule:
       interval: monthly
-    target-branch: "rel/0.12"
+    target-branch: "rel/0.13"
```
Comment (SteveYurongSu) [inline .github/dependabot.yml:64]: What about the branch rel/0.12?

Comment (HTHou) [reply .github/dependabot.yml:64]: We usually maintain two major branches. For now, they should be 0.13 and 0.14. I think 0.12 can be a big-fix branch only and it's not necessary to check its dependencies. 

How do you think?

Comment (SteveYurongSu) [reply .github/dependabot.yml:64]: IMO, if we are still maintaining the rel/0.12 branch, we'd better check the dependencies for security and safety.

Comment (HTHou) [reply .github/dependabot.yml:77]: Fixed

**Review by SteveYurongSu — APPROVED** [review]

**Pull request merged**